examples/systemcalls/systemcalls.c

```diff
@@ -10,13 +22,10 @@
 bool do_system(const char *cmd)
 {
 
-/*
- * TODO  add your code here
- *  Call the system() function with the command set in the cmd
- *   and return a boolean true if the system() call completed with success
- *   or false() if it returned a failure
-*/
-
+    int ret;
+    ret=system(cmd);
+    if (WIFSIGNALED (ret) && (WTERMSIG (ret) == SIGINT || WTERMSIG (ret) == SIGQUIT))
```

Comment:
@Sona-Subair are these the correct checks for the return conditions outlined in the description?  Look at `WIFIEXITED` and `WEXITSTATUS` instead.